Example: Exchange files in browser is broken - no `ipfs` module

<!--
Thank you for reporting an issue

This issue tracker is for bugs and issues found within the JavaScript implementation of IPFS.
If you require more general support please file an issue on our discuss forum. https://discuss.ipfs.io/

Please fill in as much of the template below as you're able.

Version: output of `jsipfs version --all` if using the CLI or `ipfs.version((err, version) => {})` if using the instance.
Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows). If using in a Browser, please share the browser version as well.
Subsystem: if known, please specify affected core module name (e.g Bitswap, libp2p, etc).

If possible, please provide code that demonstrates the problem, keeping it as
simple and free of external dependencies as you are able.
-->

- **Version**: https://github.com/ipfs/js-ipfs/commit/c1ee2474b08928ad64dac9148b7852f6c272387e


<!-- Bug, Feature, Question, Enhancement, Etc -->
#### Type: Bug

<!-- 
One of following:
  Critical - System crash, application panic.
  High - The main functionality of the application does not work, API breakage, repo format breakage, etc.
  Medium - A non-essential functionality does not work, performance issues, etc.
  Low - An optional functionality does not work.
  Very Low - Translation or documentation mistake. Something that won't give anyone a bad day.
-->
#### Severity: Medium

#### Description:

Following the installation and run instructions of that tutorial results in the following error

```
❯ npm run bundle

> exchange-files-browser@0.0.0 bundle /Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser
> browserify public/app.js > public/bundle.js

Error: Cannot find module 'ipfs' from '/Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser/public'
    at /Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Users/pascalprecht/projects/ipfs/js-ipfs/examples/exchange-files-in-browser/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:158:21)
```

Looking at the installed node_modules, the `ipfs` package is indeed missing.

#### Steps to reproduce the error:

```
$ npm install
$ npm run bundle
```

--- a/examples/exchange-files-in-browser/package.json
+++ b/examples/exchange-files-in-browser/package.json
@@ -12,7 +12,7 @@
     "http-server": "^0.11.1"
   },
   "dependencies": {
-    "ipfs": "0.29.0",
+    "ipfs": "0.30.0",
     "stream-buffers": "^3.0.1"
   }
 }

